admin/views/hdf/declaration-form.ejs

```diff
@@ -24,14 +24,20 @@
         <% if (!hdfEligibility) { %>
             <span class="govuk-heading-m">Currently unavailable</span>
             <p class="govuk-body">
-                Access to this section will become available when all pupils have completed the official live check or have a reason for not taking the check.
-                Go to <a href="/pupil-status">pupil status</a> to check these and follow the steps required.
+              You cannot complete the HDF until all pupils have completed the check or have a reason recorded for not taking the check.
+              You can view the status of your pupils and advise on next steps to take on the <a href="/pupil-status">See how many pupils have completed the official check</a> page.
+              Headteachers must submit the headteacher's declaration form once all pupils have completed the check, and by no later than 5pm on Friday 24 June.
```

Comment:
We could optionally look to make this end date dynamic - it is in the database.

Reply:
request was to keep these changes as simple as possible, and potentially avoid dynamic/logic based changes.